integration-tests/ci-visibility.spec.js

```diff
@@ -423,7 +424,8 @@ testFrameworks.forEach(({
     it('can run tests and report spans', (done) => {
       receiver.setInfoResponse({ endpoints: [] })
       receiver.payloadReceived(({ url }) => url === '/v0.4/traces').then(({ payload }) => {
-        const testSpans = payload.flatMap(trace => trace)
+        // TODO confirm this change with @juan
+        const testSpans = payload.flatMap(trace => trace).filter(span => span.type !== 'system')
```

Comment:
do we know where these spans are coming from? Is it child processes being spawned within the test frameworks or something else?

Reply:
Yes, they can come from child processes because for each one we're creating a new span which its type is 'system' so that is the reason to filter them.